docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -733,6 +991,131 @@ kubectl delete statefulset web
 statefulset "web" deleted
 ```
 
+## Pod Management Policy
+
+For some distributed systems, the StatefulSet ordering guarantees are 
+unnecessary and/or undesirable. These systems require only uniqueness and 
+identity. To address this, in Kubernetes 1.7, we introduced 
+`.spec.podManagementPolicy` to the StatefulSet API Object. 
+
+### OrderedReady Pod Management
+
+`OrderedReady` pod management is the default for StatefulSets. It tell the 
+StatefulSet controller to respect the the ordering guarantees demonstrated 
+above.
+
+### Parallel Pod Management
+
+`Parallel` pod management tells the StatefulSet controller to launch or 
+terminate all Pods in parallel, and to not wait for Pods to becoming Running 
+and Ready or completely terminated prior to launching or terminating another 
+Pod.
+
+{% include code.html language="yaml" file="web.yaml" ghlink="/docs/tutorials/stateful-application/webp.yaml" %}
+
+Download the example above, and save it to a file named `webp.yaml`
+
+This manifest is identical to the one you 
+[downloaded above](creating-a-statefulset) except that the `.spec.podManagementPolicy` 
+of the `web` StatefulSet is set to `Parallel`.
+
+In one terminal, watch the Pods in the StatefulSet.
```

Comment:
This example is feels pretty similar to the previous example with the default podManagementPolicy.  Does it add much in terms of the user's learning?